src/control/cmd/dmg/pretty/firmware.go

```diff
@@ -52,15 +52,21 @@ const (
 	nvmeSectionHeader = "NVMe Device Firmware"
 )
 
+func printScmModule(module *storage.ScmModule, out io.Writer, opts ...PrintConfigOption) error {
+	_, err := fmt.Fprintf(out, "%s\n", module.String())
```

Comment:
Should we be using Error writer here  like below?

Reply:
Oh, I see. Well, the point of ErrorWriter is that you don't need to check for an error after every write to it. If any write fails, all subsequent writes are no-ops and then the original error is returned. In this case, there was only one write so no need to bother with the wrapper.